# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../udev/udevadm-compat.patch
# Copyright (C) 2008 - 2009 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- ./udev/udevadm.c.orig	2008-09-10 01:17:59.000000000 +0200
+++ ./udev/udevadm.c	2008-09-18 10:14:14.000000000 +0200
@@ -165,9 +165,11 @@
 			if (len > 0) {
 				prog[len] = '\0';
+#if 0
 				fprintf(stderr, "the program '%s' called '%s', it should use 'udevadm %s <options>', "
				       "this will stop working in a future release\n", prog, argv[0], command);
 				err(udev, "the program '%s' called '%s', it should use 'udevadm %s <options>', "
 				    "this will stop working in a future release\n", prog, argv[0], command);
+#endif
 			}
 			rc = run_command(udev, &cmds[i], argc, argv);
 			goto out;
